app/frontend/src/components/Avatar.tsx

```diff
@@ -6,13 +6,17 @@ import { makeStyles } from "@material-ui/core/styles";
 import React from "react";
 import { User } from "../pb/api_pb";
 
-const useStyles = makeStyles({ root: {} });
+const useStyles = makeStyles({
+  root: { height: "3em", width: "3em", fontSize: "unset" },
```

Comment:
I'm not sure about this - "em" feels like a leaky abstraction to me because its effect is multiplicative/cascades and so its height is dependent on where they are in the DOM. This makes the size of the component quite unpredictable especially if elements higher up are also using flexible values.

Might be better to use `rem` instead and scale this depending on root font size?

Reply:
Yes, great idea.